fabtests/hpcs/src/test/generic.c

```diff
@@ -0,0 +1,256 @@
+/*
+ * Copyright (c) 2017-2018 Intel Corporation. All rights reserved.
+ *
+ * This software is available to you under a choice of one of two
+ * licenses.  You may choose to be licensed under the terms of the GNU
+ * General Public License (GPL) Version 2, available from the file
+ * COPYING in the main directory of this source tree, or the
+ * BSD license below:
+ *
+ *     Redistribution and use in source and binary forms, with or
+ *     without modification, are permitted provided that the following
+ *     conditions are met:
+ *
+ *      - Redistributions of source code must retain the above
+ *        copyright notice, this list of conditions and the following
+ *        disclaimer.
+ *
+ *      - Redistributions in binary form must reproduce the above
+ *        copyright notice, this list of conditions and the following
+ *        disclaimer in the documentation and/or other materials
+ *        provided with the distribution.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+ * BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#include <limits.h>
+#include <string.h>
+#include <stdio.h>
+#include <errno.h>
+#define __STDC_FORMAT_MACROS
+#include <inttypes.h>
+#include <unistd.h>
+
+#include <rdma/fi_errno.h>
+#include <rdma/fi_eq.h>
+
+#include <test/user.h>
+
+struct test_arguments {
+};
+
+void test_generic_free_arguments(struct test_arguments *arguments)
+{
+	free (arguments);
+}
+
+void test_generic_tx_init_buffer(const struct test_arguments *arguments,
+		uint8_t *buffer, size_t len)
+{
+	memset(buffer, UINT8_MAX, len);
+}
+
+void test_generic_rx_init_buffer(const struct test_arguments *arguments,
+		uint8_t *buffer, size_t len)
+{
+	memset(buffer, 0, len);
+}
```

Comment:
All these 1 line functions should be killed.  Most of the calls in this file are covered by the common fabtests functions.